nucypher/utilities/json_metrics_export.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import unicode_literals
+from prometheus_client.utils import floatToGoString
+from twisted.web.resource import Resource
+from prometheus_client.registry import REGISTRY
+import json
+
+
+class JSONMetricsResource(Resource):
+    """
+    Twisted ``Resource`` that serves metrics in JSON.
+    """
+    isLeaf = True
+
+    def __init__(self, registry=REGISTRY):
+        self.registry = registry
+
+    def render_GET(self, request):
+        request.setHeader(b'Content-Type', "text/json")
+        return generate_latest_json(self.registry)
+
+
+def generate_latest_json(registry):
```

Comment:
This is the heart of the matter!

This function is dense - will be easier to test and maintain if this was broken up into ~3 helper functions.

Reply:
Fixed